coins/src/adapters/tokenMapping.json

```diff
@@ -234,6 +234,41 @@
       "decimals": "18",
       "symbol": "TARA",
       "to": "coingecko#taraxa"
+    },
+    "0x5d0fa4c5668e5809c83c95a7cef3a9dd7c68d4fe": {
+      "decimals": "18",
+      "symbol": "WTARA",
+      "to": "asset#taraxa:0x5d0fa4c5668e5809c83c95a7cef3a9dd7c68d4fe"
+    },
+    "0x69D411CbF6dBaD54Bfe36f81d0a39922625bC78c": {
```

Comment:
sorry, dont think you understand the purpose of this file. You are linking the tokens to themselves, this file is for mapping them to other tokens/coingecko ids whose price we know

Reply:
Oh, damn. Apologies @g1nt0ki . I thought this is routing price feeds for tokens listed on our DEX. We've noticed that under https://defillama.com/protocol/taraswap#tvl-charts we have only two tokens listed, which are USDM and TARA. How could we add here all the other tokens that are listed on the DEX with their particular prices so that they show up here?

Reply:
hi @DevZi1la , now we are pulling major token prices from your dex: https://github.com/DefiLlama/defillama-server/commit/2a91bf8a73ea5002f45acddb16a172a754f741b3